client/diplodlg.cpp

```diff
@@ -880,10 +879,6 @@ void handle_diplomacy_init_meeting(int counterpart, int initiated_from)
   hmid = qMax(1, hmid);
   p.drawPixmap(1, hmid, *pix2);
   p.end();
-  queen()->sw_diplo->setIcon(QIcon(*pix3));
```

Comment:
I think we can also remove anything related to `pix3` since it wasn't used anywhere else (right?). It looks like `pix` and `pix2` have also become useless.

Reply:
Aren't they used for the diplomacy screen?

Reply:
At least `pix2` and `pix3` are deleted immediately after this code so I don't think they're used